### DISCUSSION TITLE
Spam avoidance

### DISCUSSION THREAD

**Discussion by nigeltrego** [globaleaks/globaleaks-whistleblowing-software#4053] [Ideas]
### Proposal

We are still getting spam entries albeit fewer than previous releases.  I assure that these spam entries are bot generated as the content is always the same.  Would it be possible to have the option for a Captcha style varificatation prior to report submission?  There are  open source solutions out there such as Altcha. - https://github.com/altcha-org/altcha, mosparo- https://github.com/mosparo/mosparo, Ironcaptcha - https://github.com/fabianwennink/IconCaptcha-PHP.  Thanks

### Motivation and context

I am getting spam reports that seem to have the same content so I expect that they are machine generated.

**Comment by evilaliv3** [comment]
Thank you @nigeltrego for you feedback.

First important thing: please try to use the ticketing system properly looking at existing tickets especially if you have already opened a ticket on the matter.
The project is supported by community efforts, try to collaborate efficiently or you may risk to just add additional burden.

Regarding the matter:
1. GlobaLeaks implements an automatic proof of work of the kind of ALTCHA. We can definitely raise the complexity of calculation but probably it will just slow down the number of reports.
2. The other captchas are more effective but actually if you look for the other tickets where we discussed the topic we consider that capchas in general are very very easy to be defeated and they just add complexity to real users.  In addition there are no captcha solutions that keep into consideration proper internationalization that in the context of GlobaLeaks is fundamental.

This said if you have any suggestion for improving topic 1. or 2. please feel free to proivide them or propose a pull request.
